incubator/hnc/config/crd/patches/webhook_in_hierarchyconfigurations.yaml

```diff
@@ -0,0 +1,27 @@
+# The following patch enables conversion webhook for CRD
```

Comment:
You wrote this by hand? Or was this automatically generated?

Reply:
I copied it from the previously-unused webhook_in_hierarchies.yaml (renamed and updated into webhook_in_hncconfigurations.yaml), which I believe is the scaffold from the kubebuilder tutorial for CRD conversion webhook patch.

Reply:
Ah. Where did that comment for caBundle come from again?

Reply:
It's all copied from the "webhook_in_hierarchies.yaml" that I copied the file and updated it to 3 CRDs we have.

Reply:
The versions are added by me and I commented it in the kustomization.yaml where these patches are included too.

Reply:
Ok, thanks! Those comments are out-of-date but we'll fix them another day :)